etc/flac_test/Makefile

```diff
@@ -0,0 +1,10 @@
+
+CXX := g++
+CXXFLAGS := -O0 -g -fPIC -I/home/kisner/software/conda/envs/toast/include
```

Comment:
Did you intend to commit this file?

Reply:
Yes, the `etc/` directory contains small accessory test codes which are useful to keep around as an archive, but are not directly included in the package.